custom/icds_reports/utils/aggregation_helpers/distributed/aww_incentive.py

```diff
@@ -60,25 +60,25 @@ def aggregation_query(self):
             awcm.awc_name,
             awcm.aww_name,
             awcm.contact_phone_number,
-            awcm.wer_weighed,
-            awcm.wer_eligible,
+            awcm.wer_weighed_0_2,
```

Comment:
should this use a feature flag?

Reply:
Hey, This report will be generated next month on 13, and the QA was planned to be wrapped up by 28 of this month. Therefore, I did it this way as it felt safe.